spec/runtime/load_spec.rb

```diff
@@ -56,19 +57,6 @@
         Bundler.load
       end.to raise_error(Bundler::GemfileNotFound, /omg\.rb/)
     end
-
-    it "does not find a Gemfile above the testing directory" do
```

Comment:
why is this removed?

Reply:
This is part of https://github.com/bundler/bundler/pull/7138, can you review that PR instead for the rationale? That one is ready for review :+1:.